README.md

```diff
@@ -1,9 +1,4 @@
-## sandboxedFs
-
-[![npm version](https://badge.fury.io/js/sandboxed-fs.svg)](https://badge.fury.io/js/sandboxed-fs)
-[![Build Status](https://travis-ci.org/metarhia/sandboxed-fs.svg?branch=master)](https://travis-ci.org/metarhia/sandboxed-fs)
-[![NPM Downloads/Month](https://img.shields.io/npm/dm/sandboxed-fs.svg)](https://www.npmjs.com/package/sandboxed-fs)
-[![NPM Downloads](https://img.shields.io/npm/dt/sandboxed-fs.svg)](https://www.npmjs.com/package/sandboxed-fs)
+## sandboxed-fs [![npm version](https://img.shields.io/npm/v/sandboxed-fs.svg?style=flat)](https://www.npmjs.com/package/sandboxed-fs) [![Build Status](https://travis-ci.org/metarhia/sandboxed-fs.svg?branch=master)](https://travis-ci.org/metarhia/sandboxed-fs) [![NPM Downloads/Month](https://img.shields.io/npm/dm/sandboxed-fs.svg)](https://www.npmjs.com/package/sandboxed-fs) [![NPM Downloads](https://img.shields.io/npm/dt/sandboxed-fs.svg)](https://www.npmjs.com/package/sandboxed-fs) [![license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/metarhia/sandboxed-fs/blob/master/LICENSE)
```

Comment:
Wrap the line please, there's no need to put all of the badges on the same line.

Reply:
If we will place all badges at the separate line we will see:
![Screenshot from 2020-03-20 20-22-07](https://user-images.githubusercontent.com/4405297/77194785-41146000-6ae9-11ea-861f-b0d91f4df018.png)
If we leave one badge at header line and all other at the separate:
![Screenshot from 2020-03-20 20-23-10](https://user-images.githubusercontent.com/4405297/77194922-7a4cd000-6ae9-11ea-81e5-4a72bc3b55ea.png)
Single long header line with all badges will look like:
![Screenshot from 2020-03-20 20-23-56](https://user-images.githubusercontent.com/4405297/77194998-994b6200-6ae9-11ea-8732-77ea01b6b9a2.png)

Reply:
I don't see the need to put all of the badges in the header, but if it's required, it can be done with wrapping using the `<h1>` tags.

Reply:
Ok, I changed all badges to html too because github doesn't support md inside html.